core/src/main/java/org/apache/calcite/sql/SqlOperator.java

```diff
@@ -891,6 +893,15 @@ public SqlReturnTypeInference getReturnTypeInference() {
     return returnTypeInference;
   }
 
+  /**
+   * @return the {@link Strong.Policy} strategy for this operator, or null if
+   * the policy inference is implemented by a subclass override
+   * @see Strong
+   */
+  public Supplier<Strong.Policy> getStrongPolicyInference() {
+    return null;
+  }
+
```

Comment:
`@return` -> `Returns`
`@see Strong` should have a blank line in front
Why name it `getStrongPolicyInference` ? There is no explicit interface named a policy inference.

Reply:
Thanks for your comments @danny0405 , I have fixed the javadoc.